.github/workflows/docker_release.yml

```diff
@@ -0,0 +1,69 @@
+name: One App Docker Release
+on:
+  push:
+    tags:
+      - '*'
```

Comment:
Yes that is true, i tried looking at some options and got this https://stackoverflow.com/a/60245892/5122442 but it might not cover all the options. Happy to update this though, what do you think of this?

Reply:
the suggested pattern looks good to me, however not sure if it would work for prerelease version tags like "5.0.0-rc1" is there a way we can test it?

Reply:
something like `[vV][0-9]+\.[0-9]+\S*` would allow for pre-release, we may want also add a check for the patch version

Reply:
this would require some testing

Reply:
does the action require a complete match or partial match ?

Reply:
this is not blocking for this PR we can limit this in an additional PR

Reply:
Thanks @JAdshead  i'll add it as part of the feedback work after this. This feature would only be used if we decided to push a tag manually to this repo.